test/Microsoft.Azure.WebJobs.Host.EndToEndTests/AsyncChainEndToEndTests.cs

```diff
@@ -499,6 +499,13 @@ public static void DisabledJob([QueueTrigger(Queue1Name)] string message)
             CloudBlobClient blobClient = _storageAccount.CreateCloudBlobClient();
             CloudBlobContainer container = blobClient.GetContainerReference(_resolver.ResolveInString(ContainerName));
             CloudBlockBlob blob = container.GetBlockBlobReference(NonWebJobsBlobName);
+
+            // The blob may not have committed before this function started.
+            while (!await blob.ExistsAsync())
```

Comment:
Rather than polling like this, I think the scenario of this test would remain as intended if the writer was changed from a collector binding to using an output binding. It only writes a single message anyways. That way things will happen in the correct order - when the message is received, we know the blob is ready for processing.

Reply:
Yeah good point. Done.